calcom/cal.com
# fix: Update stripe customer email when a user updates cal.com email

## What does this PR do?


When a user update their email the change should be reflected on stripe billing portal.

- Fixes https://github.com/calcom/cal.com/issues/15232
- Fixes [CAL-3817]



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



- Update your email and check that the change is reflected in stripe billing portal
